### ISSUE TITLE
using several names for host

### ISSUE THREAD

**Issue by bax-** [h2o/h2o#754]
Is it possible to use several names for one host?
e.g. If I want to use several hostnames as aliases, I would prefer doing this:
```
hosts:
  ['www.example.com', 'subdomain.example.com', 'www.example2.org']:
    listen:
      port: 80
    paths:
      "/":
        […]
```

Instead of rewriting all the settings for each host.

**Comment by bax-** [comment]
This would also come in handy if one certificate is used for several domains as aliases.

**Comment by kazuho** [comment]
> Is it possible to use several names for one host?.

No. You should use YAML alias for the purpose (see https://h2o.examp1e.net/configure/syntax_and_structure.html#yaml_alias).
